data/phi0474/phi031/phi0474.phi031.perseus-lat1.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <TEI xmlns="http://www.tei-c.org/ns/1.0">
-	<teiHeader xml:lang="eng" type="text">
+	<teiHeader xml:lang="eng">
 		<fileDesc>
 			<titleStmt>
 				<title>In Defense of Titus Annius Milo </title>
```

Comment:
The title should be Latin, yes? (whatever appears on the print page).

Reply:
See the text I did yesterday for these points. 

Subtitle <title>Pro Milone, Pro Marcello, ...</title> and <title type="sub">Machine readable text</title> are not needed. Strike these.

n="Cic." not needed. strike.

<editor role="editor" n="OCT">Albert Clark</editor> s/b <editor>Albert Curtis Clark</editor> (this is how his name appears on the page. and editor role editor is unnecessary)

funder n="org:NEH">National Endowment For The Humanities (NEH)</funder> s/b funder n="org:NEH">National Endowment for the Humanities</funder> (we don't capitalize it this way — again, trying to be consistent with existing collection)

The whole <monogr> section should read like the one I did yesterday. (So there are issues with the languages, with the title, missing scan, etc.)